src/sysv/systemd/sssd.service.in

```diff
@@ -10,10 +10,10 @@ StartLimitBurst=5
 [Service]
 Environment=DEBUG_LOGGER=--logger=files
 EnvironmentFile=-@environment_file@
-ExecStartPre=+-/bin/chown -f @SSSD_USER@:@SSSD_USER@ @sssdconfdir@
```

Comment:
There is also chown in
https://github.com/SSSD/sssd/blob/master/contrib/sssd.spec.in#L1140